universal-application-tool-0.0.1/app/views/StyleUtils.java

```diff
@@ -0,0 +1,23 @@
+package views;
+
+import com.google.common.collect.ImmutableList;
+import java.util.stream.Collectors;
+
+public class StyleUtils {
+  public static String FOCUS = "focus";
+  public static String HOVER = "hover";
+
+  public static String RESPONSIVE_SM = "sm";
+  public static String RESPONSIVE_MD = "md";
+  public static String RESPONSIVE_LG = "lg";
+  public static String RESPONSIVE_XL = "xl";
+  public static String RESPONSIVE_2XL = "2xl";
+
+  public static String applyUtilityClass(String utility, String style) {
+    return utility + ":" + style;
+  }
```

Comment:
question (non-blocking): thoughts on methods for the utilities? e.g. 

```java
public static String focus(String style) {
  return applyUtilityClass(FOCUS, style);
}
```

Reply:
I'm all about making code shorter where possible and

 `StyleUtils.focus(style)`
  is both shorter and more readable than `StyleUtils.applyUtilityClass(StyleUtils.FOCUS, style)`
  
For now I'll add `focus` and `hover`. I'm not sure that it makes sense for the responsive utils so I'll keep the current methods around for those and any shortcuts we may want later.